irods/manager/user_manager.py

```diff
@@ -82,6 +82,40 @@ def temp_password_for_user(self, user_name):
             return obf.create_temp_password(msg.stringToHashWith, conn.account.password)
 
 
+    class StoredPasswordNotEdited(RuntimeError):
+        def __init__(self,msg=None):
+            super(StoredPasswordNotEdited,self).__init__(msg)
+
+
+    def modify_own_password(self, old_value, new_value, modify_secrets_file = True):
```

Comment:
consider different function name?   `own` already means something else...    perhaps... modify_current_password?   hmm...

just a suggestion - ignore at will

Reply:
Ah, good point. Though it does need to convey somehow that the user authenticated in the session is the one (ie "me") is the whose password is being changed. Perhaps: `change_my_password`( ... )

Reply:
agreed.

alternatively `change_users_own_password` could also work.

Reply:
I'd prefer change_my_password not because it's better or anything, but because the normal usage for this would this otherwise:

  `session . users . change_users_own_password ( ... )`

... and maybe also because change_my_password is succinct and doesn't contain the word "own".

Reply:
or just `change_password`?

including words like `my` and `own` makes me think a user in PRC is represented and managed via an object handle. for example:
```python
the_user = session.users.get('kory')
the_user.change_my_password('new password')
```
those words also feel a little weird in a function name.

alternatively, based on the code in the previous response, you could use:
```python
session.users.change_user_password(the_user, password_arg_0, ..., password_arg_N)
```

Reply:
fair. let's make it `change_password` .

Reply:
Regarding your final comment, @korydraughn , the user here is understood to be the one active in the session

Reply:
fine with `change_password`

if we allow the admin to change a different user's password... that would have a better/good name too.

Reply:
@trel currently the rodsadmin can do that, but the mechanism is different:  
```
myUserObject.modify('password','newpass')
```
or:
```
sess.users.modify('myUser','password','newpass')
```

Reply:
gotcha - leave those alone then since they're already shipped.